Chap_API_Storage.tex

```diff
@@ -59,70 +59,70 @@ \section{Storage support constants}
+\declareconstitemvalueProvisional{PMIX_STORAGE_ACCESSIBILITY_CLUSTER}{0x0000000000000010}
 The storage system resources are accessible within the same cluster.
 %
-\declareconstitemProvisional{PMIX_STORAGE_ACCESSIBILITY_REMOTE}
+\declareconstitemvalueProvisional{PMIX_STORAGE_ACCESSIBILITY_REMOTE}{0x0000000000000020}
 The storage system resources are remote.
 %
 \end{constantdesc}
 
 \declarestruct{pmix_storage_persistence_t}
 \provisionalMarker{}
 
-The \refstruct{pmix_storage_persistence_t} type specifies different levels of persistence for a particular storage system.
+The \refstruct{pmix_storage_persistence_t} is a \code{uint64_t} type that defines a set of bit-mask flags for specifying different levels of persistence for a particular storage system.
 
 \begin{constantdesc}
 %
-\declareconstitemProvisional{PMIX_STORAGE_PERSISTENCE_TEMPORARY}
+\declareconstitemvalueProvisional{PMIX_STORAGE_PERSISTENCE_TEMPORARY}{0x0000000000000001}
 Data on the storage system is persisted only temporarily (i.e, it does not survive across sessions or node reboots).
 %
-\declareconstitemProvisional{PMIX_STORAGE_PERSISTENCE_NODE}
+\declareconstitemvalueProvisional{PMIX_STORAGE_PERSISTENCE_NODE}{0x0000000000000002}
 Data on the storage system is persisted on the node.
 %
-\declareconstitemProvisional{PMIX_STORAGE_PERSISTENCE_SESSION}
+\declareconstitemvalueProvisional{PMIX_STORAGE_PERSISTENCE_SESSION}{0x0000000000000004}
 Data on the storage system is persisted for the duration of the session.
 %
-\declareconstitemProvisional{PMIX_STORAGE_PERSISTENCE_JOB}
+\declareconstitemvalueProvisional{PMIX_STORAGE_PERSISTENCE_JOB}{0x0000000000000008}
 Data on the storage system is persisted for the duration of the job.
 %
-\declareconstitemProvisional{PMIX_STORAGE_PERSISTENCE_SCRATCH}
+\declareconstitemvalueProvisional{PMIX_STORAGE_PERSISTENCE_SCRATCH}{0x0000000000000010}
 Data on the storage system is persisted according to scratch storage policies (short-term storage, typically persisted for days to weeks).
 %
-\declareconstitemProvisional{PMIX_STORAGE_PERSISTENCE_PROJECT}
+\declareconstitemvalueProvisional{PMIX_STORAGE_PERSISTENCE_PROJECT}{0x0000000000000020}
 Data on the storage system is persisted according to project storage policies (long-term storage, typically persisted for the duration of a project).
 %
-\declareconstitemProvisional{PMIX_STORAGE_PERSISTENCE_ARCHIVE}
+\declareconstitemvalueProvisional{PMIX_STORAGE_PERSISTENCE_ARCHIVE}{0x0000000000000040}
 Data on the storage system is persisted according to archive storage policies (long-term storage, typically persisted indefinitely).
 %
 \end{constantdesc}
 
 \declarestruct{pmix_storage_access_type_t}
 \provisionalMarker{}
 
-The \refstruct{pmix_storage_access_type_t} type specifies different storage system access types.
+The \refstruct{pmix_storage_access_type_t} is a \code{uint16_t} type that defines a set of bit-mask flags for specifying different storage system access types.
 
```

Comment:
I think calling this out as a bitmask is fine, but if anyone is reviewing this this one is at least somewhat interesting compared to all the other constant declarations because we didn't specify this was a bitmask before and now we are.  I don't see a problem though.